apps/api/src/app/widgets/usecases/get-notifications-feed/get-notifications-feed.usecase.ts

```diff
@@ -79,22 +81,13 @@ export class GetNotificationsFeed {
         seen: command.query.seen,
         read: command.query.read,
       },
-      { limit: COUNT_LIMIT }
-      /*
-       * todo NV-2161 in version 0.16
-       *  update option as below,
-       *  update below:  hasMore = feed.length < totalCount
-       *  remove totalCount
-       * { skip: command.page * command.limit, limit: command.limit + 1 }
-       */
+      { limit: COUNT_LIMIT, skip }
     );
 
-    const hasMore = this.getHasMore(command.page, command.limit, feed, totalCount);
-
     return {
       data: feed || [],
-      totalCount: totalCount || 0,
-      hasMore,
+      totalCount: Math.min(skip + totalCount, 100),
+      hasMore: feed.length < totalCount,
```

Comment:
Could we get here `feed=null`?

Maybe we can add else on 
`    if (feed.length) `
and do return there, one request less to the db 🫣